app/Models/UserConfiguration.php

```diff
@@ -107,4 +107,16 @@ public static function default(): FreshRSS_UserConfiguration {
 		}
 		return $default_user_conf;
 	}
+
+	public function __get(string $key): mixed {
+		if ($key !== 'language') {
+			return $this->param($key);
```

Comment:
This function is deprecated. It would be nice to find an approach not involving retuning a `mixed` if possible.

Reply:
P.S. Can also wait for a follow-up PR